src/AWS.Deploy.ServerMode.Client/RestAPI.cs

```diff
@@ -1579,12 +1661,12 @@ public partial class GetConfigSettingResourcesOutput
     
     }
     
-    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.4.1.0 (Newtonsoft.Json v12.0.0.0)")]
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.4.1.0 (Newtonsoft.Json v13.0.0.0)")]
     public partial class GetDeploymentDetailsOutput 
     {
-        /// <summary>The CloudFormation Stack ID</summary>
-        [Newtonsoft.Json.JsonProperty("stackId", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
-        public string StackId { get; set; }
+        /// <summary>The CloudApplication Name</summary>
+        [Newtonsoft.Json.JsonProperty("cloudApplicationName", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public string CloudApplicationName { get; set; }
```

Comment:
Is this change related to the shutdown feature? If not it should be moved to its own PR

Reply:
Still trying to track down what happened here, the actual change to the model was made in https://github.com/aws/aws-dotnet-deploy/commit/8d395c2542aa489693e759421d345b829cfa7d04 but appears it was not included in the generated `RestAPI`

Reply:
We extracted this change and released it separately as part of https://github.com/aws/aws-dotnet-deploy/releases/tag/0.38.3, `RestAPI` should contain only changes for this PR now.